src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -65,6 +55,34 @@ class FileList extends React.Component {
     }
   }
 
+  ensureActiveItemVisible() {
+    if (!this.activeFileItem) {
+      return
+    }
+
+    const currentClassName = this.damFileList.current.className;
+
+    computeScrollIntoView(this.activeFileItem, { scrollMode: 'if-needed' }).forEach((action) => {
+      if (action.el.className !== currentClassName) {
```

Comment:
@digitalartcrew comment here would be good for those who come after us, something like "computeScrollIntoView thinks we need to scroll more elements than is actually necessary/desired"